msprime/trees.py

```diff
@@ -327,7 +323,10 @@ def draw(self):
         for u in self._tree.nodes():
             v = self._tree.get_parent(u)
             x = self._x_coords[u], self._y_coords[u]
-            dwg.add(dwg.circle(center=x, r=3))
+            if u in self._node_colours and self._node_colours[u] is not None:
```

Comment:
There was another reason, when I was percolating colours (or absence of them) through nodes, and it was useful to have a flag to mark a default colour setting. But more generally, I dislike the code bombing out when I do something I would expect to work. As a user, I would hope that setting something to None would be covered in a sensible way (and indeed would expect it from an external library). But you seem to have a different approach, whereby the code drops out more often than I, as a user, would hope. Either way it's your codebase, though. Just a user's perspective.